perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -81,6 +82,8 @@ sub send() {
 
             # this is for the new_updates alerts
             next if $row->{non_public} and $row->{user_id} != $row->{alert_user_id};
+            # do not sent triage comments
+            next if $row->{item_extra} && $row->{item_extra} =~ /triage_report,I1:1/;
```

Comment:
Would it not make more sense to mark the comment as already alerted on at the time it was created? Then no need to check it here.